### PR TITLE
modified Last Updated timestamp format

### LINKED ISSUE #141: Last Updated timestamp in a weird format
The use of M-D-Y formatted dates trips me up every time I look at the last updated timestamp. Please change to ISO format, like we use on the coding progress dashboard, or display the date in my computer's locale.

### PR DESCRIPTION
**Pull request by DanielMwendwa**
Closes: #141

### CODE REVIEW

File: public/graph.js [hunk 1]
```diff
@@ -763,7 +766,7 @@ class GraphController {
             }
         });
 
-        let fullDateFormat = d3.timeFormat("%c");
+        let fullDateFormat = d3.timeFormat("%d %b %Y %H:%M:%S %p");
```
Comment (IsaackMwenda) [inline public/graph.js]: make this consistent with message coding table last updated time.

**Review by IsaackMwenda — CHANGES_REQUESTED** [review]

File: public/UI.js [hunk 2]
```diff
@@ -54,7 +54,8 @@ class UIController {
             while (statusBody.firstChild) {
                 statusBody.removeChild(statusBody.firstChild);
             }
-            let lastUpdate = data["last_update"];
+            let fullDateFormat = d3.timeFormat("%d %b %Y %H:%M:%S %p"),
```
Comment (IsaackMwenda) [inline public/UI.js]: Hi @DanielMwendwa I meant to make the graph.js last updated to be consistent with coding progress Last updated: 2020-02-13 06:25:04

**Review by IsaackMwenda — CHANGES_REQUESTED** [review]
The pushed branch does not contain the changes please fix and request review.

**Comment by DanielMwendwa** [comment]
@IsaackMwenda please check PR #163

**Pull request closed without merging**